.github/workflows/go.yml

```diff
@@ -65,8 +65,15 @@ jobs:
         with:
           go-version: "${{env.GO_VERSION}}"
 
-      - name: Build Docker image
-        run: make update-devel-image
+      - name: Build Forwarder
+        run: go build -o forwarder ./cmd/forwarder
```

Comment:
Use make build instead.

Reply:
I didn't use makefile, because it was broken and I forgot about it. Here's the fix https://github.com/saucelabs/forwarder/pull/277. I should've done it in the first place.